app/models/space.rb

```diff
@@ -27,6 +27,21 @@ def reviews_for_facility(facility)
     AggregatedFacilityReview.find_by(space: self, facility: facility)
   end
 
+  def facilities_in_category(category)
+    AggregatedFacilityReview
+      .where(space: self)
+      .includes(:facility)
+      .where(facilities: { facility_category: category })
+      .map do |result|
```

Comment:
You don't need two `.where` doing 
```ruby
    AggregatedFacilityReview
      .includes(:facility)
      .where(space: self, facilities: { facility_category: category })
      .map do |result|
```
will compile to the same query, but it may be clearer to have them seperate?

Reply:
(Thinking that can be done in a seperate PR, if we want to. Merging this one now)

Reply:
I don't think getting the tooltip is possible without later translating them in the UI, as the select query is directly mapped to SQL and will not be able to call ruby methods